modules/fscloud/README.md

```diff
@@ -20,6 +20,40 @@ Important: In order to avoid unexpected breakage in the account against which th
 ## Note
 The services 'compliance', 'directlink', 'iam-groups', 'containers-kubernetes', 'user-management' does not support restriction per location.
 
+### Usage
+
+```hcl
+module "cbr_fscloud" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/fscloud"
+  version          = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
```

Comment:
The code block here probably fails the linter pre-commit hook. Suggest you indent the code so it passes a linter

Reply:
It was not failing though, one of the } was missing, I added it.